sales/sales.js

```diff
@@ -1,99 +1,83 @@
-var express = require('express');
-var http = require('http');
-var os = require('os');
+'use strict';
 
-var salesData = require('./data');
-var consul = require('./consul');
+const Consulite = require('consulite');
+const Express = require('express');
+const Customers = require('./lib/customers');
+const Data = require('./lib/data');
 
-var app = express();
-var customersHosts = [];
+const app = Express();
```

Comment:
This is to demonstrate that the framework doesn't matter. I also want to add another service that uses hapi + mongo.